quickstart/static_files/training.py

```
@@ -0,0 +1,11 @@
+from featureform import Client
+
+client = Client(insecure=True)
+
+ts_name = "fraud_training"
+ts_variant = "quickstart"
+
+dataset = client.training_set(ts_name, ts_variant)
+
+for i, batch in enumerate(dataset):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)